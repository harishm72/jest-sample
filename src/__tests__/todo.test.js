import React from 'react';
import {shallow} from 'enzyme';
import '../setupTests'
import Todo from '../todo';
import renderer from 'react-test-renderer'

describe('Todo --- Snapshot',()=>{
    it('Rednders the App Component', () => {
        const renderedValue =  renderer.create(<Todo />).toJSON()
        expect(renderedValue).toMatchSnapshot();
    });
  });

describe("Todo shallow render" , () => {

    let wrapper
    beforeEach(() => {
        wrapper = shallow(<Todo/>);
    })
    
})


// it('renders welcome message', () => {
//   const wrapper = shallow(<App/>);
//   expect(wrapper).toContainReact('BrowserRouter');
// });