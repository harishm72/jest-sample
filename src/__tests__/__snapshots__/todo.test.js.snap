// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo --- Snapshot Rednders the App Component 1`] = `
<div>
  <form
    className="App"
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      value=""
    />
    <button>
      Submit
    </button>
  </form>
  <ul />
</div>
`;
